### DISCUSSION TITLE
zustand for vue

### DISCUSSION THREAD

**Discussion by AwesomeDevin** [pmndrs/zustand#1464] [Ideas]
I want to commit a pr to support for vue, should I do it ?

**Comment by dai-shi** [comment]
I'd suggest to create a separate package and list it in https://github.com/pmndrs/zustand/blob/main/docs/integrations/3rd-party-libraries.md

**Reply by AwesomeDevin** [reply]
tks,  good idea

**Comment by Zikoat** [comment]
This library (https://github.com/wobsoriano/vue-zustand) has been added to the 3rd party libraries, so this discussion should be set as answered and closed.

I have also worked on vue 2 support here: https://gist.github.com/Zikoat/ec47ff3646f889d09f8c6d350e6060f6
